get_github_coderabbit_review/process_pr_comments.py

```
@@ -0,0 +1,225 @@
+def process_pr_comments():
+    """プルリクエストのコメントを処理してAIエージェント向け形式で出力"""
+
+    # スクリプトファイル基準のベースディレクトリを設定
+    base_dir = Path(__file__).resolve().parent
+    
+    # データファイルを読み込み
+    try:
+        pr_data_path = base_dir / 'pr_104_raw_data.json'
+        with open(pr_data_path, 'r', encoding='utf-8') as f:
+            pr_data = json.load(f)
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)